docs/frontend/overview.rst

```diff
@@ -0,0 +1,172 @@
+Overview
+========
+
+The adhocracy 3 frontend is an opinionated web framework based on
```

Comment:
But what is it then? It is definitely not an application. Maybe a set of applications with a common core?